client/state/reader/follows/reducer.js

```diff
@@ -11,10 +11,76 @@ import {
 	READER_FOLLOW,
 	READER_UNFOLLOW,
 	READER_FOLLOWS_RECEIVE,
+	READER_SUBSCRIBE_TO_NEW_POST_EMAIL,
+	READER_SUBSCRIBE_TO_NEW_COMMENT_EMAIL,
+	READER_UPDATE_NEW_POST_EMAIL_SUBSCRIPTION,
+	READER_UNSUBSCRIBE_TO_NEW_POST_EMAIL,
+	READER_UNSUBSCRIBE_TO_NEW_COMMENT_EMAIL,
 } from 'state/action-types';
 import { prepareComparableUrl } from './utils';
 import { createReducer } from 'state/utils';
 
+function updatePostSubscription( state, payload, shouldSkipUpdate, newProps ) {
```

Comment:
Can we add a jsdoc here? I had the same question about `shouldSkipUpdate` as @samouri below (https://github.com/Automattic/wp-calypso/pull/12703/files#r111429552), so probably worth documenting.